app/src/main/res/values-ar/strings.xml

```diff
@@ -10,6 +10,7 @@
   <string name="menu_help">مساعدة</string>
   <string name="exploration_activity_title">مشغل رحلة الاستكشاف</string>
   <string name="help_activity_title">المساعدة</string>
+  <string name="bottom_sheet_options_menu_close">إغلاق</string>
```

Comment:
Please don't add strings for other languages, that's a separate process.

Reply:
Removed